desk/app/channels.hoon

```diff
@@ -136,7 +138,7 @@
           [/v3 %channel-response-4 ~]
           [/v3/said %channel-said-1 %channel-denied ~]
         ::
-          [/v4/said %channel-said-2 %channel-denied ~]
+          [/v4/said %channel-said-3 %channel-said-2 %channel-denied ~]
```

Comment:
PR description says we're _adding_ `/v4/said`, but this and the scry endpoint look like they were pre-existing.

Unless we're 100% certain these endpoints weren't used in published versions of the app, we shouldn't be changing the marks that gets emitted by them, and instead add new endpoints.

For this watch specifically, multiple different marks/versions doesn't fit into our usual pattern. There's a version in the path, so it should give a fixed set of marks.